### PR TITLE
[OAP-1631][Doc]Update MLlib Doc for OAP 1.0

### PR DESCRIPTION
**Pull request by HongW2019**
## What changes were proposed in this pull request?
Fix: #1631

### CODE REVIEW

**Comment by HongW2019** [comment]
@xwu99 Please have a review and help merge it.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
 # Optimized Analytics Package for Spark\* Platform (OAP)
+LEGAL NOTICE: Your use of this software and any required dependent software (the "Software Package") is subject to the terms and conditions of the software license agreements for the Software Package, which may also include notices, disclaimers, or license terms for third party or open source software included in or with the Software Package, and your use indicates your acceptance of all such terms. Please refer to the "TPP.txt" or other similarly-named text file included with the Software Package for additional details.
 
-OAP is a project to optimize Spark by providing optimized implementation of packages for various aspects including cache, shuffle, and so on. In this version, we include the optimized implementation of SQL Index and Data Source Cache supporting DRAM and PMem, RDD Cache PMem Extension, Shuffle Remote PMem Extension and Remote Shuffle.
+Optimized Analytics Package for Spark* Platform is under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0).
```
Comment (xwu99) [inline README.md:4]: You can highlight Apache 2.0 license.

**Review by xwu99 — CHANGES_REQUESTED** [review]

File: oap-mllib/README.md [hunk 2]
```diff
@@ -11,9 +11,19 @@ For those algorithms that are not accelerated by Intel MLlib, the original Spark
...
-After downloaded, you can refer to the following [Running](#Running) section to try out.
+You can get a pre-built intel MLlib jar to get started, the jar can be downloaded from [OAP-jars-tarball](https://github.com/Intel-bigdata/OAP/releases/download/v0.9.0-spark-3.0.0/oap-0.9.0-bin-spark-3.0.0.tar.gz) then extracting this tarball. 
+
+Then you can refer to the following [Running](#Running) section to try out.
+
+You can also build the package from source code, please refer to [Building](#Building) section.
+
+### Python/PySpark Users Preferred
+
+You can use a pre-built JAR to get started, if you have finished [OAP-Installation-Guide](../docs/OAP-Installation-Guide.md), you can find compiled Intel MLlib jar `oap-mllib-x.x.x-with-spark-x.x.x.jar` in `/root/miniconda2/envs/oapenv/oap_jars/`.
```
Comment (xwu99) [inline oap-mllib/README.md:24]: It's bad practice to use /root to install any of the software. it's better to use $HOME/miniconda2/envs/oapenv/oap_jars as in current user home directory.

Comment (xwu99) [reply oap-mllib/README.md:24]: The principle is using root when only necessary. Install software to system-wide when only necessary.

Comment (HongW2019) [reply oap-mllib/README.md:24]: @yao531441 Can we change the  /root to get a better practice?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,14 @@
 # Optimized Analytics Package for Spark\* Platform (OAP)
+LEGAL NOTICE: Your use of this software and any required dependent software (the "Software Package") is subject to the terms and conditions of the software license agreements for the Software Package, which may also include notices, disclaimers, or license terms for third party or open source software included in or with the Software Package, and your use indicates your acceptance of all such terms. Please refer to the "TPP.txt" or other similarly-named text file included with the Software Package for additional details.
```
[the remark this answers is not in the document]
Comment (HongW2019) [reply README.md]: Have modified the font of Legal part to differentiate from OAP introduction. As for its location , intel Legal request "You must include at the top of the README file included with your distribution ......"

File: oap-mllib/README.md (hunk 2 above)
Comment (yao531441) [reply oap-mllib/README.md]: @xwu99 Ok,we will recommend  usrs not to use the 'root' to install Conda.

**Review by xwu99 — CHANGES_REQUESTED** [review]

File: docs/OAP-Installation-Guide.md [hunk 4]
```diff
@@ -55,11 +57,11 @@ sh dev/install-runtime-dependencies.sh
...
-spark.executor.extraLibraryPath /root/miniconda2/envs/oapenv/lib/
-spark.driver.extraLibraryPath /root/miniconda2/envs/oapenv/lib/
-spark.executor.extraClassPath      /root/miniconda2/envs/oapenv/oap_jars/$OAP_FEATURE.jar
-spark.driver.extraClassPath      /root/miniconda2/envs/oapenv/oap_jars/$OAP_FEATURE.jar
+spark.executorEnv.LD_LIBRARY_PATH $HOME/miniconda2/envs/oapenv/lib/
+spark.executor.extraLibraryPath $HOME/miniconda2/envs/oapenv/lib/
+spark.driver.extraLibraryPath $HOME/miniconda2/envs/oapenv/lib/
+spark.executor.extraClassPath      $HOME/miniconda2/envs/oapenv/oap_jars/$OAP_FEATURE.jar
+spark.driver.extraClassPath      $HOME/miniconda2/envs/oapenv/oap_jars/$OAP_FEATURE.jar
 ```
```
[the remark this answers is not in the document]
Comment (xwu99) [reply docs/OAP-Installation-Guide.md:65]: no need to add extra / after a path, the same below.

[the remark this answers is not in the document]
Comment (HongW2019) [reply docs/OAP-Installation-Guide.md:63]: Thanks for detailed suggestions.

File: docs/OAP-Installation-Guide.md [hunk 5]
```diff
@@ -35,13 +35,15 @@ conda create -n oapenv -y python=3.7
 conda activate oapenv
```
[the remark this answers is not in the document]
Comment (HongW2019) [reply docs/OAP-Installation-Guide.md]: Added.

File: README.md [hunk 6]
```diff
@@ -1,10 +1,15 @@
...
+##### \* LEGAL NOTICE: Your use of this software and any required dependent software (the "Software Package") is subject to the terms and conditions of the software license agreements for the Software Package, which may also include notices, disclaimers, or license terms for third party or open source software included in or with the Software Package, and your use indicates your acceptance of all such terms. Please refer to the "TPP.txt" or other similarly-named text file included with the Software Package for additional details.
+
+##### \* Optimized Analytics Package for Spark* Platform is under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0).
+
+OAP is a project to optimize Spark by providing optimized implementation of packages for various aspects including cache, shuffle, native sql engine, Mllib and so on. In this version, OAP contains the optimized implementations of SQL Index and Data Source Cache supporting DRAM and PMem, RDD Cache PMem Extension, Shuffle Remote PMem Extension,
```
Comment (xwu99) [inline README.md:7]: sql => SQL

File: docs/OAP-Installation-Guide.md [hunk 7]
```diff
@@ -15,9 +15,9 @@ We have tested OAP on Fedora 29 and CentOS 7.6 (kernel-4.18.16). We recommend yo
 - **Conda Requirements**   
 Install Conda on your cluster nodes with below commands and follow the prompts on the installer screens.:
 ```bash
-wget -c https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh
-chmod 777 Miniconda2-latest-Linux-x86_64.sh 
-bash Miniconda2-latest-Linux-x86_64.sh 
+$ wget -c https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh
+$ chmod 777 Miniconda2-latest-Linux-x86_64.sh
```
Comment (xwu99) [inline docs/OAP-Installation-Guide.md:19]: no need to to set extra permissions, just chmod +x

File: docs/OAP-Installation-Guide.md [hunk 8]
```diff
@@ -15,9 +15,9 @@ We have tested OAP on Fedora 29 and CentOS 7.6 (kernel-4.18.16). We recommend yo
...
 Install Conda on your cluster nodes with below commands and follow the prompts on the installer screens.:
 ```bash
-wget -c https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh
-chmod 777 Miniconda2-latest-Linux-x86_64.sh 
-bash Miniconda2-latest-Linux-x86_64.sh 
+$ wget -c https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh
+$ chmod 777 Miniconda2-latest-Linux-x86_64.sh 
+$ bash Miniconda2-latest-Linux-x86_64.sh 
 ```
 For changes to take effect, close and re-open your current shell.To test your installation,  run the command `conda list` in your terminal window. A list of installed packages appears if it has been installed correctly.
```
Comment (xwu99) [inline docs/OAP-Installation-Guide.md:22]: an extra space. your current shell.To => your current shell. To

**Review by xwu99 — CHANGES_REQUESTED** [review]

**Comment by xwu99** [comment]
@HongW2019 thanks for your effort! merged!

**Pull request merged**